src/serlo-editor-integration/create-plugins.tsx

```diff
@@ -71,7 +73,21 @@ export function createPlugins({
   return [
     {
       type: EditorPluginType.Text,
-      plugin: createTextPlugin({ serloLinkSearch: instance === Instance.De }),
+      plugin: createTextPlugin({
+        serloLinkSearch: instance === Instance.De,
+        formattingOptions: [
+          TextEditorFormattingOption.code,
+          TextEditorFormattingOption.colors,
+          TextEditorFormattingOption.headings,
+          TextEditorFormattingOption.katex,
+          TextEditorFormattingOption.links,
+          TextEditorFormattingOption.lists,
+          TextEditorFormattingOption.math,
+          TextEditorFormattingOption.paragraphs,
+          TextEditorFormattingOption.richTextBold,
+          TextEditorFormattingOption.richTextItalic,
```

Comment:
can we use could the `defaultFormattingOptions` here?

Reply:
Simplified the configuration